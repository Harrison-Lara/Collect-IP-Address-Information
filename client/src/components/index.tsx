export * from './textfield/textfield.component';
export * from './button/button.component';