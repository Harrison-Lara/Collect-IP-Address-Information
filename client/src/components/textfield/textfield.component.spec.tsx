import React from 'react';
import { TextField } from 'components';
import { shallow } from 'enzyme';

describe('TextField', () => {
    const onChange = jest.fn();

    it('Render a textfield', () => {
        const component = shallow(<TextField id="test" label='testing' onChange={onChange} />);

        expect(component).toHaveLength(1);
        expect(component).toMatchSnapshot();
    });
});