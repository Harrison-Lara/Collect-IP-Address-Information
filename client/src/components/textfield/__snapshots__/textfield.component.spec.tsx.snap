// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextField Render a textfield 1`] = `ShallowWrapper {}`;
